index.html

```diff
@@ -9399,7 +9504,7 @@ <h2>Translatable States and Properties</h2>
 	</section>
 	<section id="global_states">
 		<h2>Global States and Properties</h2>
-		<p>Some <a>states</a> and <a>properties</a> are applicable to all host language <a>elements</a> regardless of whether a <a>role</a> is applied. The following global states and properties are supported by all roles and by all base markup elements.</p>
+		<p>Some <a>states</a> and <a>properties</a> are applicable to all host language <a>elements</a> regardless of whether a <a>role</a> is applied. The following global states and properties are supported by all roles and by all base markup elements unless otherwise prohibited. Any global states and properties which are prohibited on any role will be listed in the Prohibited States and Properties section.</p>
```

Comment:
>Any global states and properties which are prohibited on any role will be listed in the Prohibited States and Properties section.

This sounds contradictory. The second "on any" could be interpreted to mean "from all," meaning that the state or property cannot be used at all on any element. In addition, the statement is future tense instead of present tense. And, the location of listing is not clear. Suggest:

>If a role prohibits use of any global states or properties, those states or properties are listed as prohibited in the characteristics table included in the section that defines the role.